dmutils/formats.py

```diff
@@ -12,30 +12,59 @@
+    """
+    "timeformat" is used in summary tables where space is tight and times are shown on their own line.
+    Currently (Jan 2018) only used in admin app tables.
+    """
     return _format_date(value, default_value, DISPLAY_TIME_FORMAT)
 
 
 def shortdateformat(value, default_value=None):
+    """
+    "shortdateformat" was designed for use in summary tables where space is tight and dates are shown on their own line.
+    The original intended use was in conjunction with "timeformat" in admin app summary tables.
+    It is now (Jan 2018) also used in briefs-frontend on the "Publish your requirements" page only.
+
+    ** USE OUR STANDARD dateformat RATHER THAN THIS UNLESS THERE IS A GOOD REASON NOT TO **
+    """
     return _format_date(value, default_value, DISPLAY_SHORT_DATE_FORMAT, localize=False)
 
 
 def dateformat(value, default_value=None):
+    """
+    "dateformat" is the standard format used for dates in our frontend apps.
+    If you're displaying a date on a page then this is probably the one you want.
+    Examples of use:
+      * dates that opportunities are created/published/closing/closed/withdrawn
+      * dates that framework communication files were published
+      * dates of revisions made to services (shown to admin users)
+    """
     return _format_date(value, default_value, DISPLAY_DATE_FORMAT, localize=False)
 
 
 def datetimeformat(value, default_value=None, localize=True):
-    # en_GB locale uses uppercase AM/PM which contravenes our style guide
+    """
+    "datetimeformat" is the standard format used for timestamps (i.e. date and time) in our frontend apps.
+    If you're displaying a date and time on a page then this is probably the one you want (unless it is a
+    brief closing date, in which case use "utcdatetimeformat" below).
+    Examples of use:
+      * dates that opportunities are created/published/closing/closed/withdrawn
+      * dates that framework communication files were published
+      * dates of revisions made to services (shown to admin users)
+    """
     formatted_date = _format_date(value, default_value, DISPLAY_DATETIME_FORMAT, localize=localize)
     if formatted_date:
+        # en_GB locale uses uppercase AM/PM which contravenes our style guide
         return formatted_date.replace('AM', 'am').replace('PM', 'pm').replace(" 0", " ")
     return formatted_date
 
 
 def utcdatetimeformat(value, default_value=None):
     """
-    Use this filter to format deadline timestamps that are stored in the database as
+    Only use this filter to format deadline timestamps that are stored in the database as
     23:59:59 (UTC+00), when the localisation would otherwise display the date as 12.59am
-     on the next day (due to daylight savings), potentially causing confusion for buyers/suppliers.
+    on the next day (due to daylight savings), potentially causing confusion for buyers/suppliers.
+    This is currently (Jan 2018) used wherever we show the closing date for a brief.
```

Comment:
It's also used for the Clarification Questions deadline I think? (At least it should be!)

Reply:
Yep, I can add that.